### PR TITLE
Use structured Event for migration performance

### PR DESCRIPTION
**Pull request by ecotopian**
Use SPStructured Event instead of Timing event as this can be easier visualized in Looker

### CODE REVIEW

File: Client/Ecosia/Analytics/Analytics.swift [hunk 1]
```diff
@@ -261,10 +261,11 @@ final class Analytics {
     }
 
     func migrated(_ migration: Migration, in seconds: TimeInterval) {
-        tracker.track(SPTiming.build({
-            $0.setCategory("migration")
-            $0.setVariable(migration.rawValue)
-            $0.setTiming(Int(seconds * 1000))
+        tracker.track(SPStructured.build({
+            $0.setCategory(Category.migration.rawValue)
```
Comment (vauxhall) [inline Client/Ecosia/Analytics/Analytics.swift:265]: lovely

File: Client/Ecosia/Analytics/Analytics.swift [hunk 2]
```diff
@@ -261,10 +261,11 @@ final class Analytics {
...
+            $0.setAction(Action.completed.rawValue)
+            $0.setLabel(migration.rawValue)
+            $0.setValue(seconds * 1000)
```
Comment (vauxhall) [inline Client/Ecosia/Analytics/Analytics.swift:268]: are we interested in milliseconds also?

**Review by vauxhall — APPROVED** [review]
🚀

File: Client/Ecosia/Analytics/Analytics.swift (hunk 2 above)
Comment (ecotopian) [reply Client/Ecosia/Analytics/Analytics.swift:268]: yes - the time event had `Int` as type so I went for milliseconds to have some accuracy.. let's stick to that for now 👍

**Pull request merged**